src/components/Hero/SearchView/SearchView.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from "react";
+import { Logo } from "../../Logo/Logo";
+import { Button } from "../../UI/Button/Button";
+import { ButtonType } from "../../UI/Button/Button.consts";
+import { Search } from "../../UI/Search/Search";
+import "./SearchView.scss";
+import { ISearchViewProps } from "./SearchView.types";
+
+export const SearchView: FC<ISearchViewProps> = ({ toggleModal }) => {
+  return (
+    <>
+      <div className="search-view__top-banner">
+        <Logo />
+        <Button variant={ButtonType.secondary} onClick={toggleModal}>
+          + ADD MOVIE
+        </Button>
+      </div>
+      <section className="search-view__content">
+        <h1 className="search__heading">FIND YOUR MOVIE</h1>
+        <Search setQuery={() => {}} onSearch={() => {}} />
```

Comment:
According to a task you have to implement filtering (search by name)

Reply:
Just checked one more time specs and it wasn't mentioned anywhere:

3 - All data fetches are implemented as actions and received from store by components
4 - Sorting by release date and rating is implemented as redux actions
5 - Filtering by genre is implemented as redux actions

so that's why it's missing - although I have just done it so implementation is done in next PR